src/main/java/com/hotel/pad/services/HotelAvailabilityService.java

```diff
@@ -0,0 +1,33 @@
+package com.hotel.pad.services;
+
+
+import com.hotel.pad.client.HotelAvailabilityClient;
+import com.hotel.pad.mappers.AvailRequestSegmentsMapper;
+import com.hotel.pad.mappers.PosMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.opentravel.ota._2003._05.OTAHotelAvailRQ;
+import org.opentravel.ota._2003._05.OTAHotelAvailRS;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class HotelAvailabilityService {
+
+    public void getAvailableHotelItemsFromSupplier(){
+        HotelAvailabilityClient client = new HotelAvailabilityClient();
+        OTAHotelAvailRQ hotelAvailRQ = new OTAHotelAvailRQ();
+        PosMapper posMapper = new PosMapper();
```

Comment:
give a constructor and add HotelAvailabityclient and posmaper as arguments instead of initialising here